src/index.js

```diff
@@ -0,0 +1,12 @@
+import './style.css';
+import Tasklist from './modules/Tasklist.js';
+
+window.addEventListener('load', () => {
+  const tasklist1 = new Tasklist();
+  tasklist1.retrievedata();
+
+  tasklist1.renderdata();
+
```

Comment:
Although, separate lines of code might improve the clarity of a project or functions like this one, consider avoiding leaving empty lines behind within functions. It is not a good practice and can make your code look "smelly"

Reply:
ok